docs/source/explore/get-started/install.rst

```diff
@@ -53,70 +76,81 @@ To install on Windows, simply double-click on the downloaded installer file to o
-2. Uninstall any versions of Vircadia that are visible (Including any Steam installs). 
-3. Once Vircadia is uninstalled, browse to your %Program Files% directory. Delete all folders related to Vircadia. If you installed through Steam, these folders will be located at ``C:/Program Files(x86)/Steam/steamapps/common``. 
+1. Click on the Start menu and type "Add or Remove Programs" in your Windows search bar.
+2. Uninstall any versions of Vircadia that are visible (Including any Steam installs).
+3. Once Vircadia is uninstalled, browse to your %Program Files% directory. Delete all folders related to Vircadia. If you installed through Steam, these folders will be located at ``C:/Program Files(x86)/Steam/steamapps/common``.
 
-.. warning:: 
+.. warning::
 
-    The next 3 steps will permanently delete your Sandbox content. If you wish to keep this content, copy %AppData%/Local/Project Athena/assignment-client to another location on your computer before proceeding. Repeat for %AppData%/Roaming/Project Athena/assignment-client. 
+    The next 3 steps will permanently delete your Sandbox content. If you wish to keep this content, copy %AppData%/Local/Project Athena/assignment-client to another location on your computer before proceeding. Repeat for %AppData%/Roaming/Project Athena/assignment-client.
 
-4. Browse to your local %AppData% folder (usually ``C:/Users/<your_username>/AppData/Local``). If you do not see the folder, make sure you can view hidden folders. In File Explorer, click View and make sure "Hidden Items" is checked. Delete all folders related to Vircadia.  
-5. Browse to your roaming %AppData% folder (usually ``C:/Users/<your_username>/AppData/Roaming``). Delete all folders related to Vircadia.  
-6. Re-install Vircadia using the steps above. To restore your Sandbox content, copy the ``assignment-client`` folders you backed up back to their respective locations.  
+4. Browse to your local %AppData% folder (usually ``C:/Users/<your_username>/AppData/Local``). If you do not see the folder, make sure you can view hidden folders. In File Explorer, click View and make sure "Hidden Items" is checked. Delete all folders related to Vircadia.
+5. Browse to your roaming %AppData% folder (usually ``C:/Users/<your_username>/AppData/Roaming``). Delete all folders related to Vircadia.
+6. Re-install Vircadia using the steps above. To restore your Sandbox content, copy the ``assignment-client`` folders you backed up back to their respective locations.
 
 ^^^^^^^^^^^^^^^^^^^^^^^^
 Mac Clean Install
 ^^^^^^^^^^^^^^^^^^^^^^^^
 
-1. Open your Applications folder and delete the Vircadia folder. 
+1. Open your Applications folder and delete the Vircadia folder.
 2. Open the ``<username>/.config`` folder. This is a hidden folder than is accessible by going to *Go > Home*. Press the keyboard shortcut ``Command + Shift + .`` (period).
-3. Delete the ``highfidelity.io`` folder. 
+3. Delete the ``highfidelity.io`` folder.
```

Comment:
I don't recall if I fixed this as of yet. So for now we'll leave that reference I suppose.